.jshintrc

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Yes, our experiment has been successful. Please take linting config from this file: https://github.com/fluid-project/infusion/blob/master/.eslintrc.json and surrounding package.json and Gruntfile.js